packages/styles/scss/patterns/sub-patterns/content-section/_content-section.scss

```diff
@@ -30,16 +30,33 @@
       }
     }
     &__heading {
-      margin-top: $carbon--layout-01;
+      margin-top: $carbon--layout-03;
       margin-bottom: $carbon--layout-05;
 
       @include type-style('expressive-heading-02');
+
+      @include carbon--breakpoint('md') {
+        margin-top: $carbon--layout-03;
+      }
+      @include carbon--breakpoint('lg') {
+        margin-top: $carbon--layout-01;
+      }
     }
     &__children {
       @include carbon--breakpoint(lg) {
         @include carbon--make-col(12, 16);
       }
     }
+    .#{$prefix}--card {
+      &__eyebrow {
+        margin-top: $carbon--layout-01;
+      }
+      &__heading {
+        @include carbon--breakpoint(lg) {
+          margin-bottom: $carbon--layout-03;
+        }
+      }
+    }
```

Comment:
Didn't this get solved in #1556? Have you pulled down the latest yet? Also, I think if this is still an issue we'd want to solve this at the card level most likely.

Reply:
yes @photodow I have latest and I got your spacing PR changes , #1478 has spacing issue in small break point, when I added 
`@include carbon--breakpoint('sm') {
  margin-top:  $carbon--layout-03;
}` and it does nothing so made the above changes.

Reply:
Looking at the latest [here](http://ibmdotcom-react.mybluemix.net/iframe.html?id=patterns-sections-cardsectionimages--default) at the smallest breakpoint it looks correct to me.

![image](https://user-images.githubusercontent.com/3793636/75904154-4d6d9d00-5e08-11ea-8d43-0b3ec2d4f7a7.png)

Reply:
I think @photodow is right, the spacing seems to be fixed after the global changes from #1556 
We must have missed closing this one when those were applied?

Reply:
yeah, @oliviaflory I close this PR. thx @photodow